ADotNet.Infrastructure.Build/Program.cs

```diff
@@ -35,6 +36,18 @@ static void Main(string[] args)
                     }
                 },
 
+                EnvironmentVariables = new Dictionary<string, string>
+                {
+                    {
```

Comment:
Can you move that value calculation to a method to tidy up this section?

```suggestion
    { "IS_RELEASE_CANDIDATE", IsReleaseCandidate() }
```